types/index.d.ts

```diff
@@ -58,15 +76,49 @@ interface BaseOptions<T extends JSToPostgresTypeMap> {
     value?: (value: any) => any;
     /** Transforms entire rows */
     row?: (row: postgres.Row) => any;
+  } | {
+    /** Transforms incoming column names */
+    column?: {
+      /** SQL to JS */
+      from?: (column: string) => string;
+      /** JS to SQL */
+      to?: (column: string) => string;
+    };
+    /** Transforms incoming row values */
+    value?: {
+      /** SQL to JS */
+      from?: (value: unknown) => any;
+      // /** JS to SQL */
+      // to?: (value: unknown) => any; // unused
+    };
+    /** Transforms entire rows */
+    row?: {
+      /** SQL to JS */
+      from?: (row: postgres.Row) => any;
+      // /** JS to SQL */
+      // to?: (row: postgres.Row) => any; // unused
+    };
   };
   /** Connection parameters */
   connection: Partial<postgres.ConnectionParameters>;
+  /**
+   * Use 'read-write' with multiple hosts to ensure only connecting to primary
+   * @default process.env['PGTARGETSESSIONATTRS']
+   */
+  target_session_attrs: undefined | 'read-write';
```

Comment:
could also be one of `'read-only' | 'primary' | 'standby' | 'prefer-standby'`

Reply:
That's weird, it's already fixed... :thinking: Here is the in latest commit (43d78d5):

https://github.com/porsager/postgres/blob/43d78d548694dea7643b98c434f139ca4c18fdbf/types/index.d.ts#L101

Your comment just disappeared from the Files changed menu, maybe it's a Github cache issue? :smile:

Reply:
Haha.. I think I might have gone to the previous commit and commented there, and so I didn't see you'd already fixed it 🙈